addons/mail/static/src/core/common/settings_model.js

```diff
@@ -77,6 +78,16 @@ export class Settings extends Record {
     edgeBlurAmount = 10;
     showOnlyVideo = false;
     useBlur = false;
+    blurPerformanceWarning = fields.Attr(false, {
+        compute() {
+            const rtc = this.store?.rtc;
+            if (!rtc || !this.useBlur) {
+                return false;
+            }
+            return this.useBlur && rtc.state?.cameraTrack && !hasHardwareAcceleration();
+        },
+        eager: true,
```

Comment:
No need to `eager: true` it: this is used by a template with `t-if`, let it compute itself automatically when needed.